Admin/Admin.php

```diff
@@ -2540,8 +2546,12 @@ public function id($entity)
     /**
      * {@inheritdoc}
      */
-    public function setValidator(ValidatorInterface $validator)
+    public function setValidator($validator)
```

Comment:
Ouch! It is true... I don't know but maybe there is a way to avoid this BC and solve the problem...